src/EditorFeatures/CSharpTest/CodeActions/EnableNullable/EnableNullableTests.cs

```diff
@@ -26,7 +27,7 @@ public class EnableNullableTests
                 var document = project.Documents.First();
 
                 // Only the input solution contains '#nullable enable'
-                if (!document.GetTextSynchronously(CancellationToken.None).ToString().Contains("#nullable enable"))
+                if (!Regex.IsMatch(document.GetTextSynchronously(CancellationToken.None).ToString(), "#nullable  ?enable"))
```

Comment:
❓My regex fu is limited. Is this matching 0 or 1 spaces before enable? Or is the `?` applied to `enable` somehow?

Reply:
yeah, i don't quite get the need for this. afaict, this is allowing for either one or two spaces after #nullable.  Is that really needed?

Reply:
maybe ` +` or `\s+` if you want to match any amount of whitespace?

Reply:
Spaces between `#` and `nullable` are allowed by the compiler.

Reply:
> afaict, this is allowing for either one or two spaces after #nullable. Is that really needed?

Yes. All of the tests have one space except for one test that has two spaces.

Reply:
➡️ Comment is now updated

Reply:
This whole method is a workaround for the fact that the test library can't cleanly represent a code action that changes compilation options. Considering a few options and this will be updated after.